### ISSUE TITLE
[Feature Request] Character limit in body

### ISSUE THREAD

**Issue by NickeManarin** [nhevia/discord-styled-releases#1]
If I create a release with more than 2048(?) characters, the workflow executes but the message is not displayed by Discord.
It appears that these are the imposed limits:

https://discord.com/developers/docs/resources/channel#embed-limits

It would be nice if the workflow could trim down the release body to just below the limit, and perhaps add a "…".

Here's my yml:

```yml
# This is a basic workflow to help you get started with Actions
name: Discord • Releases

# Controls when the action will run. 
on: 
  # Triggers the workflow on new release.
  release:
    types: [published]
      
# A workflow run is made up of one or more jobs that can run sequentially or in parallel
jobs:
  # This workflow contains a single job called "released"
  released:
    # The type of runner that the job will run on
    runs-on: ubuntu-latest
    name: Sends release details to Discord/News channel
    
    # Steps represent a sequence of tasks that will be executed as part of the job
    steps:
      - name: Sending release notes
      
        # Checks-out a repository, to send the release notes
        uses: nhevia/discord-styled-releases@main
        
        # Gets the ID and Token from the project secrets
        with:
          webhook_id: ${{ secrets.DISCORD_WEBHOOK_ID }}
          webhook_token: ${{ secrets.DISCORD_WEBHOOK_TOKEN }}
```

**Comment by nhevia** [comment]
Thanks for the feedback! 
This is fixed in #3 adding an ellipsis with more than 1500 characters, including a link to the actual github release page. Title also has a link to the url now https://github.com/nhevia/discord-styled-releases/commit/ffc81a06e92b3eac4900769226427be93c6b46ca.

It was indeed an issue with discord embeds limits for webhook's body.
I checked the actual response from the webhook and it's returning them with line breaks (\r\n) which adds a lot of noise with these types of releases.

**Comment by NickeManarin** [comment]
Awesome! 
Thanks for quick fix :)
